backend/test_requirements.txt

```diff
@@ -0,0 +1 @@
+pytest==6.2.1
```

Comment:
By convention this file should be called: `dev-requirements.txt`
This file should also include `-r requirments.txt` at the beginning.